tests/test_transformation/test_arcsin_transformer.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+from sklearn.exceptions import NotFittedError
+
+from feature_engine.transformation import ArcsinTransformer
+
+
+def test_fit_raises_error_if_na_in_df(df_na):
+    # test case 2: when dataset contains na, fit method
+    with pytest.raises(ValueError):
+        transformer = ArcsinTransformer()
+        transformer.fit(df_na)
+
+
+def test_transform_raises_error_if_na_in_df(df_vartypes, df_na):
+    # test case 3: when dataset contains na, transform method
+    with pytest.raises(ValueError):
+        transformer = ArcsinTransformer()
+        transformer.fit(df_vartypes)
+        transformer.transform(df_na[["Name", "City", "Age", "Marks", "dob"]])
```

Comment:
this dataframe contains variables outside the 0-1 range, so like this, we would not know if it fails because of the na, or because of the variable range.

The variable Marks is between 0-1, so I think we need to set up the transformer in line 17 like this:

transformer(ArcsinTransformer(variables=["Marks"])